### PR TITLE
Added a merging audo and video command

### PR DESCRIPTION
**Pull request by Ray6464**
I need this command again and again but keep forgetting.

- [ ] The page has 8 or fewer examples.
     Not really, the page is a bit long, however people using this tool don't really care as long as it's well organized and to the point.
- [ x ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [ x ] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/ffmpeg.md [hunk 1]
```diff
@@ -34,3 +34,7 @@
 - Convert MP4 video to VP9 codec. For the best quality, use a CRF value (recommended range 15-35) and -b:video MUST be 0:
 
 `ffmpeg -i {{input_video}}.mp4 -codec:video libvpx-vp9 -crf {{30}} -b:video 0 -codec:audio libopus -vbr on -threads {{number_of_threads}} {{output_video}}.webm`
+
+- Merge audio with video. This command takes in a video file, and an audio file and give out a merged file with both audio and video:
+
+`ffmpeg -i video.mp4 -i audio.wav -c:v copy -c:a aac -strict experimental output.mp4`
```
Comment (sbrl) [inline pages/common/ffmpeg.md]: Perhaps we could put the things that can be changed in tokens?
```suggestion
`ffmpeg -i {{video}}.mp4 -i {{audio}}.wav -c:v copy -c:a aac -strict experimental {{output}}.mp4`
```

**Comment by Ray6464** [comment]
> Hey, thanks for the edit @Ray6464! I've left a comment below for you to review.
> 
> In addition, it looks like we've got too many examples on this page. Which one would you suggest is the least useful?
> 
> Finally, it looks like you haven't yet signed the CLA. We need you to sign this in order to be able to merge your PR. Note that the email address you've committed with has to be present in your GitHub account settings.

I don't know anything about video editing, however the "Convert MP4 video to VP9 codec" seems useless for me for the most part, it has 4 "unknowns" in it which suggests that if the user really using ffmpeg for that kind of stuff they should just read man pages instead or tldr, I mean 4 variables? how in the world is that tldr.

I also suggest that tldr should keep both smaller and longer versions of files that works when tldr is used with a "-L" flag. I mean I idea behind tldr is not keeping the docs small but the examples small and simple right? Keeping the docs as small as 8 examples is a huge setback without a clear philosophy of what example goes in and what goes out.

TL;DR: My commit in, lengthy commits out!

**Comment by sbrl** [comment]
My knowledge of video editing is more limited than I'd like it to be, but VP9 is the latest WebM codec (which is great for embedding videos in web pages, since it tends to generate smaller files), so I think that example is useful. However, I certainly agree that it can be simplified. For example if a specific value must be zero, then there's no need to mention it in the description.

Regarding shorter and longer versions of pages, that's probably best suggested in a separate issue. Perhaps you could open one?

**Comment by zdroid** [comment]
I also agree this example is useful. Perhaps we could change maximum number of examples to 10.

**Comment by Ray6464** [comment]
> My knowledge of video editing is more limited than I'd like it to be, but VP9 is the latest WebM codec (which is great for embedding videos in web pages, since it tends to generate smaller files), so I think that example is useful. However, I certainly agree that it can be simplified. For example if a specific value must be zero, then there's no need to mention it in the description.
> 
> Regarding shorter and longer versions of pages, that's probably best suggested in a separate issue. Perhaps you could open one?

How about this, if the example is really useful it's allowed through the 8-example limit, than later if we have a over-flow problem we can break the overflowing pages into small pages with an 8-limit and larger pages.

Also keep in mind that the 8-limit is will eventually kill this tool, it's very limiting; Instead of 8-limits, let the 8-limit be a rule of a loose thumb and lets be very strict about what goes in and what comes out. How about that?

**Comment by zdroid** [comment]
I would definitely keep the limit, @Ray6464. Our pages are supposed to be short and concise. On the other side, I'd prefer 10 examples instead of arbitrarily chosen 8.

**Comment by Ray6464** [comment]
> I would definitely keep the limit, @Ray6464. Our pages are supposed to be short and concise. On the other side, I'd prefer 10 examples instead of arbitrarily chosen 8.

Works for me!

**Comment by sbrl** [comment]
Changing the limit on the number of examples is not a small change - it should probably be discussed in a separate issue. I'm sure there was a reason for the existing limit - I'm just not sure what it is.

**Comment by zdroid** [comment]
It definitely does. Do you have some formed text for the matter, so you can open the issue yourself?

**Comment by sbrl** [comment]
Define `formed text`

**Comment by zdroid** [comment]
That means: do you have some worthy thoughts or comments about it to write down in the (supposed) issue's description?

**Comment by pxgamer** [comment]
@sbrl/@zdroid, https://github.com/tldr-pages/tldr/issues/2799

**Comment by sbrl** [comment]
Ah, thanks @pxgamer! I did have a feeling that we discussed it before.

**Comment by zdroid** [comment]
:frowning_woman:
